452 23:51:24.891806 --- Starting bitbake server pid 452 at 2025-08-29 23:51:24.891793 ---
452 23:51:24.892585 Started bitbake server pid 452
452 23:51:24.892699 Entering server connection loop
452 23:51:24.892710 Lockfile is: /home/yocto/tech_prj/Kas/bitbake.lock
Socket is /home/yocto/tech_prj/Kas/bitbake.sock (True)
452 23:51:24.893321 Accepting [<socket.socket fd=6, family=AddressFamily.AF_UNIX, type=SocketKind.SOCK_STREAM, proto=0, laddr=bitbake.sock>] ([])
452 23:51:24.893867 Processing Client
452 23:51:24.893889 Connecting Client
452 23:51:24.894072 Running command ['setFeatures', [2, 1]]
452 23:51:24.894096 Sending reply (None, None)
452 23:51:24.894134 Command Completed (socket: True)
452 23:51:24.894382 Running command ['updateConfig', {'halt': True, 'force': False, 'invalidate_stamp': None, 'dry_run': False, 'dump_signatures': [], 'extra_assume_provided': [], 'profile': False, 'prefile': [], 'postfile': [], 'server_timeout': None, 'nosetscene': False, 'setsceneonly': False, 'skipsetscene': False, 'runall': [], 'runonly': None, 'writeeventlog': None, 'build_verbose_shell': False, 'build_verbose_stdout': False, 'default_loglevel': 20, 'debug_domains': {}}, {'BB_ENV_PASSTHROUGH_ADDITIONS': 'ALL_PROXY BBPATH_EXTRA BB_LOGCONFIG BB_NO_NETWORK BB_NUMBER_THREADS BB_SETSCENE_ENFORCE BB_SRCREV_POLICY DISTRO FTPS_PROXY FTP_PROXY GIT_PROXY_COMMAND HTTPS_PROXY HTTP_PROXY MACHINE NO_PROXY PARALLEL_MAKE SCREENDIR SDKMACHINE SOCKS5_PASSWD SOCKS5_USER SSH_AGENT_PID SSH_AUTH_SOCK STAMPS_DIR TCLIBC TCMODE all_proxy ftp_proxy ftps_proxy http_proxy https_proxy no_proxy ', 'PWD': '/home/yocto/tech_prj/Kas', 'HOME': '/root', 'BBPATH': '/home/yocto/tech_prj/build', 'LC_ALL': 'en_US.UTF-8', 'PATH': '/home/yocto/layers/oe-core/scripts:/home/yocto/layers/oe-core/bitbake/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin', 'OE_ADDED_PATHS': '/home/yocto/layers/oe-core/scripts:/home/yocto/layers/oe-core/bitbake/bin:', 'HOSTNAME': 'hamza-Laptop', 'LANG': 'en_US.UTF-8', 'LS_COLORS': 'rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=00:su=37;41:sg=30;43:ca=30;41:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.tar=01;31:*.tgz=01;31:*.arc=01;31:*.arj=01;31:*.taz=01;31:*.lha=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.tlz=01;31:*.txz=01;31:*.tzo=01;31:*.t7z=01;31:*.zip=01;31:*.z=01;31:*.dz=01;31:*.gz=01;31:*.lrz=01;31:*.lz=01;31:*.lzo=01;31:*.xz=01;31:*.zst=01;31:*.tzst=01;31:*.bz2=01;31:*.bz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tz=01;31:*.deb=01;31:*.rpm=01;31:*.jar=01;31:*.war=01;31:*.ear=01;31:*.sar=01;31:*.rar=01;31:*.alz=01;31:*.ace=01;31:*.zoo=01;31:*.cpio=01;31:*.7z=01;31:*.rz=01;31:*.cab=01;31:*.wim=01;31:*.swm=01;31:*.dwm=01;31:*.esd=01;31:*.jpg=01;35:*.jpeg=01;35:*.mjpg=01;35:*.mjpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.webp=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=00;36:*.au=00;36:*.flac=00;36:*.m4a=00;36:*.mid=00;36:*.midi=00;36:*.mka=00;36:*.mp3=00;36:*.mpc=00;36:*.ogg=00;36:*.ra=00;36:*.wav=00;36:*.oga=00;36:*.opus=00;36:*.spx=00;36:*.xspf=00;36:', 'PYTHONPATH': '/home/yocto/layers/oe-core/bitbake/lib:', 'TERM': 'xterm', 'SHLVL': '1', 'BUILDDIR': '/home/yocto/tech_prj/build', 'OLDPWD': '/home/yocto/tech_prj', '_': '/home/yocto/layers/oe-core/bitbake/bin/bitbake'}, ['/home/yocto/layers/oe-core/bitbake/bin/bitbake', '-c', 'cleanstate']]
452 23:51:24.897638 Sending reply (None, 'bb.BBHandledException()\nTraceback (most recent call last):\n  File "/home/yocto/layers/oe-core/bitbake/lib/bb/cookerdata.py", line 162, in wrapped\n    return func(fn, *args)\n  File "/home/yocto/layers/oe-core/bitbake/lib/bb/cookerdata.py", line 179, in parse_config_file\n    return bb.parse.handle(fn, data, include, baseconfig=True)\n  File "/home/yocto/layers/oe-core/bitbake/lib/bb/parse/__init__.py", line 119, in handle\n    return h[\'handle\'](fn, data, include, baseconfig)\n  File "/home/yocto/layers/oe-core/bitbake/lib/bb/parse/parse_py/ConfHandler.py", line 119, in handle\n    abs_fn = resolve_file(fn, data)\n  File "/home/yocto/layers/oe-core/bitbake/lib/bb/parse/__init__.py", line 139, in resolve_file\n    raise IOError(errno.ENOENT, "file %s not found in %s" % (fn, bbpath))\nFileNotFoundError: [Errno 2] file conf/bitbake.conf not found in /home/yocto/tech_prj/build\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File "/home/yocto/layers/oe-core/bitbake/lib/bb/command.py", line 90, in runCommand\n    result = command_method(self, commandline)\n  File "/home/yocto/layers/oe-core/bitbake/lib/bb/command.py", line 290, in updateConfig\n    command.cooker.updateConfigOpts(options, environment, cmdline)\n  File "/home/yocto/layers/oe-core/bitbake/lib/bb/cooker.py", line 463, in updateConfigOpts\n    self.reset()\n  File "/home/yocto/layers/oe-core/bitbake/lib/bb/cooker.py", line 1736, in reset\n    self.initConfigurationData()\n  File "/home/yocto/layers/oe-core/bitbake/lib/bb/cooker.py", line 283, in initConfigurationData\n    self.databuilder.parseBaseConfiguration()\n  File "/home/yocto/layers/oe-core/bitbake/lib/bb/cookerdata.py", line 261, in parseBaseConfiguration\n    self.data = self.parseConfigurationFiles(self.prefiles, self.postfiles)\n  File "/home/yocto/layers/oe-core/bitbake/lib/bb/cookerdata.py", line 471, in parseConfigurationFiles\n    data = parse_config_file(os.path.join("conf", "bitbake.conf"), data)\n  File "/home/yocto/layers/oe-core/bitbake/lib/bb/cookerdata.py", line 174, in wrapped\n    raise bb.BBHandledException()\nbb.BBHandledException\n')
452 23:51:24.897676 Command Completed (socket: True)
452 23:51:25.147582 Processing Client
452 23:51:25.147710 Disconnecting Client (socket: True)
452 23:51:25.147948 No timeout, exiting.
452 23:51:25.248353 Exiting (socket: True)
452 23:51:25.250070 Exiting as we could obtain the lock
sys:1: ResourceWarning: unclosed file <_io.TextIOWrapper name='/home/yocto/tech_prj/Kas/bitbake-cookerdaemon.log' mode='a+' encoding='UTF-8'>
